docs/Using Price Feeds/get-the-latest-price.html

```diff
@@ -91,19 +91,65 @@
 
 ## Getting a different price denomination
 
-Chainlink Data Feeds for asset prices can be used in combination to derive denominated price pairs in other currencies.
+Chainlink Price Feeds can be used in combination to derive denominated price pairs in other currencies.
 
-If you require a denomination other than what is provided, you can use two feeds to derive the pair that you need. For example, if you needed an BTC / EUR price, you could take the BTC / USD feed and the EUR / USD feed and derive BTC / EUR using [OpenZeppelin's SafeMath](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/utils/math/SafeMath.sol) division.
+If you require a denomination other than what is provided, you can use two price feeds to derive the pair that you need. For example, if you needed a BTC / EUR price, you could take the BTC / USD feed and the EUR / USD feed and derive BTC / EUR using division.
 
 ![Request Model Diagram](/images/price-feed-conversion-equation.gif)
 
+> 🚧 Important
+> 
+> If you are using a Solidity version before 0.8.0 use [OpenZeppelin's SafeMath](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/utils/math/SafeMath.sol) for division.
 
-```solidity
-using SafeMath for uint256;
 
-function derivedPrice(uint256 priceBTCUSD, uint256 priceEURUSD) public view returns (uint256) {
-    return div(priceBTCUSD, priceEURUSD);
-}
+```solidity
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.7;
+
+import "@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol";
+
+contract DerivedPriceConsumerV3 {
+
+    AggregatorV3Interface internal priceFeedBtcUsd;
+    AggregatorV3Interface internal priceFeedEurUsd;
+    int public decimals;
+
+    /**
+     * Network: Kovan
+     * Aggregator: BTC/USD
+     * Address: 0x6135b13325bfC4B00278B4abC5e20bbce2D6580e
+     * Decimals: 8
+     * Aggregator: EUR/USD
+     * Address: 0x0c15Ab9A0DB086e062194c273CC79f41597Bbf13
+     * Decimals: 8
+     */
+    constructor() {
+        priceFeedBtcUsd = AggregatorV3Interface(0x6135b13325bfC4B00278B4abC5e20bbce2D6580e);
+        priceFeedEurUsd = AggregatorV3Interface(0x0c15Ab9A0DB086e062194c273CC79f41597Bbf13);
+        decimals = 10**8;
```

Comment:
Let's not hard code decimals like this. You can actually call the `decimals` function to get the number of decimals. ie:

```
priceFeedEurUsd.decimals()
```

Reply:
There are two situations where this is conversion is used:
(1) USD-base denominated (illustrated in above)
(2) blockchain base currency denominated (BNB, ETH, etc.) 

Most all conversions will be through USD denominations on both pairs which will be 8 decimals.  If it is not through a USD denominated base then you would be using multiplication.
Ex. ALPHA/BNB * BNB/USD = ALPHA/USD
ALPHA/BNB (18 decimals)
BNB/USD (8 decimals)
.002257198340965213 * 418.01476442
2257198340965213 * 41801476442
You would need to divide the result by the sum of the two decimals.  Regardless, the example in the docs should illustrate Option 1 as we would be more willing to spin up a price feed for a requested USD denomination if needed.  As for not hard coding, that's fine but should I just get the decimals of the numerator and use that?

Reply:
Chatted with Max. Going to have 2 functions to address this.